src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/MakeAnApplicationTypesProvider.java

```diff
@@ -126,6 +128,7 @@ public DynamicList getMakeAnApplicationTypes(Callback<AsylumCase> callback) {
                 values.add(new Value(LINK_OR_UNLINK.name(), LINK_OR_UNLINK.toString()));
                 values.add(new Value(JUDGE_REVIEW.name(), JUDGE_REVIEW.toString()));
                 values.add(new Value(OTHER.name(), OTHER.toString()));
+                values.add(new Value(CHANGE_HEARING_TYPE.name(), CHANGE_HEARING_TYPE.toString()));
```

Comment:
I just realized that `CHANGE_HEARING_TYPE` is missing from the `LISTING` block. It escaped me earlier when I checked the PR